wrapper/Tools/OpenMMMD.py

```diff
@@ -408,7 +408,7 @@ def setupForcefields(system, space):
 
     system.setProperty("space", space)
     system.setProperty("switchingFunction", CHARMMSwitchingFunction(cutoff_dist.val))
-    system.setProperty("CombRules", VariantProperty(combining_rules.val))
+    system.setProperty("CombiningRules", VariantProperty(combining_rules.val))
```

Comment:
Sorry - "combiningRules" with a small "c". Sorry to be picky - need to match property name in other parts of the code

Reply:
Sorry about that! My mistake. I'm fixing it now

Reply:
hi @chryswoods ! can you see the commits that I did or should I "merge the pull request" ?

Reply:
Yes, I can see the commits. Thanks for checking :-). No, policy is you can't merge your own pull request. As I've commented below, the change has introduced a compile error for OS X. Once fixed I will review again and then merge.